colibris/views.py

```diff
@@ -1,17 +1,248 @@
+import abc
+import traceback
+from json import JSONDecodeError
 
-from aiohttp import web
+from aiohttp import web, hdrs
 from aiohttp_apispec import docs
+from aiohttp_apispec import response_schema, request_schema
 
-from colibris import app
+from colibris.persist import Model
+from colibris.schemas import ModelSchema
+from marshmallow import ValidationError
+from peewee import IntegrityError
+
+from colibris import app, api
 from colibris.conf import settings
 
 
 async def home(request):
     raise web.HTTPFound(settings.API_DOCS_PATH)
 
 
+# TODO: I would suggest to move it into some template.
 @docs(tags=['Service'],
       summary='The health-check endpoint')
 async def health(request):
     h = await app.get_health()
     return web.json_response(h)
+
+
+class _GenericMixinMeta(abc.ABCMeta):
+    def __init__(cls, name, bases, attrs):
+        if getattr(cls, 'model', None) is not None:
```

Comment:
@ccrisan, this is my best shot. What's happening is to not fail for `None` values at server start, but it will fail when the `model` attribute (same for `body_schema_class` or  `query_schema_class`) is used. I checked what DRF does and it looks like it's the very same approach. Maybe we can improve latter, but now, I don't have a better option. Looks very complicated to handle "abstract" classes.

Reply:
This looks fairly decent.